dev-docs/modules/akamaiDapRtdProvider.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: page_v2
+title: Akamai DAP Real Time Data Provider Module
+display_name: Akamai DAP Real Time Data Provider Module
+description: Akamai DAP Real Time Data Provider Module
+page_type: module
+module_type: rtd
+module_code : akamaiDapRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+# Akamai DAP Real Time Data Provider Module
+{:.no_toc}
+
+* TOC
+{:toc}
+
+Akamai DAP Real time data Provider automatically invokes the DAP APIs and submit audience segments and the SAID to the bid-stream.
```

Comment:
I would request that you either define these terms (DAP and SAID) or link to a place where users can learn about it.

Reply:
Updated the readme file with (DAP and SAID) definition.